src/eduid_common/api/decorators.py

```diff
@@ -56,7 +57,7 @@ def decorated_function(*args, **kwargs):
 def require_dashboard_user(f):
```

Comment:
Shouldn't this one return a user document from the central db loaded in to a DashboardUser class?